download-data.sh

```diff
@@ -21,9 +21,18 @@ do
   curl --create-dirs $URL/$RELEASE/$file -o data/$RELEASE/$file -z data/$RELEASE/$file
 done
 
+# Download reference and gencode file from public ftp
+REFERENCE="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/GRCh38.primary_assembly.genome.fa.gz"
+GENCODE="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz"
+cd data
+curl -JO $REFERENCE
+curl -JO $GENCODE
+cd -
+
 # Check the md5s for everything we downloaded except CHANGELOG.md
 cd data/$RELEASE
 md5sum -c md5sum.txt
+mv pbta-snv-consensus_11122019.zip ../
```

Comment:
for now, the consensus zip file will be downloaded to `data/$RELEASE/` as part of the release. We can just move such file out to the `data/` after the md5 check.

Reply:
My concern with moving rather than symlinking is that having everything that is included with a data download live in the `data/$RELEASE` folder is convenient if folks want to go back and compare different versions of files to narrow down an issue they are having (#260). It's also helpful for generating files in CI because one of the inputs to that is a `data/$RELEASE` directory (#278).